client/src/containers/example/index.tsx

```diff
@@ -19,10 +21,17 @@ const Example: React.FC = () => {
 	const showNotification = (notification: TSetNotificationArgs) => {
 		dispatch(setNotificationState(notification));
 	};
+	const history = useHistory();
```

Comment:
bad. read this https://stackoverflow.com/questions/42672842/how-to-get-history-on-react-router-v4

We shouldn't have many useHistory in project. So we have historyHelper in 'client/src/helpers/'
import from there and use :)

ALSO, in react components we can just have smth like this: 
```
import React from 'react';
interface IButtonsBlockProps {}
export const ButtonsBlock = (props: IButtonsBlockProps) => {
	console.log(history);
	return (
		<div className={styles.buttonsBlock}>
			<Button className={clsx(ButtonClasses.red)}>Preview</Button>
			<div className="select">
				<Label>Go to</Label>
			</div>
		</div>
	);
};
```
Look at the 'console.log(history)', yes I've not declared it. It is already in every React component. I'm not sure if it works the same, but it should!

Reply:
@Paul-Vasylenko fixed, thank you